static/code-examples/kafka-mtls/client-intents.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: k8s.otterize.com/v1alpha3
 kind: ClientIntents
 metadata:
-  name: client
-  namespace:  otterize-tutorial-kafka-mtls
+  name: client-intents-for-client
```

Comment:
we typically recommend keeping the name here, the same as the service name. Which would make this `name: client`, and the other one `name: kafka`.